README.md

```diff
@@ -8,3 +9,12 @@ Please contribute!  Make sure to review the [pull request checklist](/CONTRIBUTI
 
 If you find rr useful, please [add a testimonial](https://github.com/mozilla/rr/wiki/Testimonials).
 
+# systems requirements
+* Intel chip with [Nehalem](https://en.wikipedia.org/wiki/Nehalem_%28microarchitecture%29) (2010) or later microarchitecture
+
+ * VM with perf counter virtualization is ok
+ * userspace: x86
```

Comment:
Ah, cool. My main source was [this presentation](https://mozilla.github.io/rr/rr.html) and it didn't mention 64bit support. I updated the readme. Thank you for pointing out the 64bit support.